nautobot/extras/models/jobs.py

```diff
@@ -844,7 +845,10 @@ class ScheduledJob(BaseModel):
     """Model representing a periodic task."""
 
     name = models.CharField(
-        max_length=200, verbose_name="Name", help_text="Human-readable description of this scheduled task", unique=True
+        max_length=CHARFIELD_MAX_LENGTH,
+        verbose_name="Name",
+        help_text="Human-readable description of this scheduled task",
+        unique=True,
     )
     task = models.CharField(
         # JOB_MAX_NAME_LENGTH is the longest permitted module name as well as the longest permitted class name,
```

Comment:
Note that `JOB_MAX_NAME_LENGTH` is just 100 characters; we *probably* need to keep this constant as-is, as `task` here has a max_length of `JOB_MAX_NAME_LENGTH + 1 + JOB_MAX_NAME_LENGTH` and is `db_index=True`; IIRC MySQL doesn't like db_index on fields over 255 characters.